packages/mobile/test/schemas.ts

```diff
@@ -342,6 +342,14 @@ export const v2Schema = {
   },
 }
 
+export const v3Schema = {
+  ...v2Schema,
+  identity: {
+    ...v2Schema.identity,
+    isFetchingAddresses: false,
+  },
+}
+
```

Comment:
You don't need a new schema version for just adding props.

Reply:
Gotcha, will add that prop to v2Schema